homework.md

```diff
@@ -4,77 +4,78 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
```

Comment:
Hi Ahmad, good job with the homework, there is one small typo.
it says:"What is the 2 _ 3 _ 6 _ 1 _ 10 * 34?"
should say: "What is the 2 * 3 * 6 * 1 * 10 * 34?
Great job!

Reply:
Hi Lonut,

Thank you for informing me, it is appreciated.

Reply:
Hi Ahmad, great work! For question 9:Could you add where the folder is saved?Github or desktop or any other place?